Update microsoft365-apps-usage.md (#2717)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/activity-reports/microsoft365-apps-usage.md

|||
 |:-----|:-----|
 |1. <br/> |The **Microsoft 365 Apps usage** report can be viewed for trends over the last 7 days, 30 days, 90 days, or 180 days. However, if you select a particular day in the report, the table (7) will show data for up to 28 days from the current date (not the date the report was generated). <br/> |

>  |2. <br/> |The data in each report usually covers up to the past 7 days. <br/> |

Reviewer: ```suggestion
 |2. <br/> |The data in each report usually covers up to the last seven days. <br/> |
```

>  |3. <br/> |The **Users** view iew shows the trend in the number of active users for each app – Outlook, Word, Excel, PowerPoint, OneNote, and Teams. "Active users" are any who perform any intentional actions within these apps. <br/> |

Reviewer: ```suggestion
 |3. <br/> |The **Users** view shows the trend in the number of active users for each app – Outlook, Word, Excel, PowerPoint, OneNote, and Teams. "Active users" are any who perform any intentional actions within these apps. <br/> |
```

 |4. <br/> |The **Platforms** view shows the trend of active users across all apps for each platform – Windows, Mac, Web, and Mobile. <br/> |

>  |5.<br/>|On the **Users** chart, the Y axis is the number of unique active users for the respective app. On the **Platforms** chart, the Y axis is the number of unique users for the respective platform. X axis on both charts is the date on which an app was used on a given platform.rm.<br/>|

Reviewer: ```suggestion
 |5.<br/>|On the **Users** chart, the Y-axis is the number of unique active users for the respective app. On the **Platforms** chart, the Y-axis is the number of unique users for the respective platform. The X-axis on both charts is the date on which an app was used on a given platform.<br/>|
```

>  6.<br/>|You can filter the series you see on the chart by selecting an item in the legend. For example, on the **Users** chart, select Outlook, Word, Excel, PowerPoint, OneDrive,or Teams to see only the info related to each one. Changing this selection doesn't change the info in the grid table below it.|

Reviewer: ```suggestion
 6.<br/>|You can filter the series you see on the chart by selecting an item in the legend. For example, on the **Users** chart, select Outlook, Word, Excel, PowerPoint, OneDrive, or Teams to see only the info related to each one. Changing this selection doesn't change the info in the grid table below it.|
```

>  |7.<br/>|The table shows you a breakdown of data at the per-user level. You can add or remove columns from the table. <br/><br/>**Username** is the email address of the user who performed the activity on Microsoft Apps.<br><br/>**Last activation date (UTC)** is the latest date on which the user activated their Microsoft 365 Apps subscription.<br/><br/>**Last activity date (UTC)** is the latest date an intentional activity was performed by the user. To see activity that occurred on a specific date, select the date directly in the chart.<br/><br/>Following columns corresponding to each app that identifies if the user was active on that app in the period selected:<br> <br>**Outlook** <br>**Word** <br>**Excel**<br>**PowerPoint** <br>**OneNote**<br><br> Following columns corresponding to each platform that identifies if the user was active on that platform for any app (within Microsoft 365 Apps) in the period selected:<br><br>**Outlook (Windows)**<br>**Outlook (Mac)**<br>**Outlook (Web)** <br>**Outlook (Mobile)**<br> **Word (Windows)**<br> **Word (Mac)**<br> **Word (Web)**<br> **Word (Mobile)**<br> **Excel (Windows)**<br> **Excel (Mac)**<br> **Excel (Web)**<br> **Excel (Mobile)**<br> **PowerPoint (Windows)**<br> **PowerPoint (Mac)**<br>**PowerPoint (Web)**<br> **PowerPoint (Mobile)**<br> **OneNote (Windows)**<br> **OneNote (Mac)**<br> **OneNote (Web)**<br>**OneNote (Mobile)**<br> **Teams (Windows)**<br> **Teams (Mac)**<br> **Teams (Web)**<br>**Teams (Mobile)** |

Reviewer: ```suggestion
 |7.<br/>|The table shows you a breakdown of data at the per-user level. You can add or remove columns from the table. <br/><br/>**Username** is the email address of the user who performed the activity on Microsoft Apps.<br><br/>**Last activation date (UTC)** is the latest date on which the user activated their Microsoft 365 Apps subscription.<br/><br/>**Last activity date (UTC)** is the latest date an intentional activity was performed by the user. To see activity that occurred on a specific date, select the date directly in the chart.<br/><br/>The following columns correspond to each app that identifies if the user was active on that app in the period selected:<br> <br>**Outlook** <br>**Word** <br>**Excel**<br>**PowerPoint** <br>**OneNote**<br><br> The following columns correspond to each platform that identifies if the user was active on that platform for any app (within Microsoft 365 Apps) in the period selected:<br><br>**Outlook (Windows)**<br>**Outlook (Mac)**<br>**Outlook (Web)** <br>**Outlook (Mobile)**<br> **Word (Windows)**<br> **Word (Mac)**<br> **Word (Web)**<br> **Word (Mobile)**<br> **Excel (Windows)**<br> **Excel (Mac)**<br> **Excel (Web)**<br> **Excel (Mobile)**<br> **PowerPoint (Windows)**<br> **PowerPoint (Mac)**<br>**PowerPoint (Web)**<br> **PowerPoint (Mobile)**<br> **OneNote (Windows)**<br> **OneNote (Mac)**<br> **OneNote (Web)**<br>**OneNote (Mobile)**<br> **Teams (Windows)**<br> **Teams (Mac)**<br> **Teams (Web)**<br>**Teams (Mobile)** |
```

Reviewer: Please change two instances of "Following columns corresponding" to "The following columns correspond".

 |8.<br/>|Select the **Manage columns** icon to add or remove columns from the report.|

>  |9.<br/>|You can also export the report data into an Excel .csv file by selecting the **Export** link. This exports data for all users and enables you to do simple aggregation, sorting and filtering for further analysis. If you have less than 100 users, you can sort and filter within the table in the report itself. If you have more than 100 users, in order to filter and sort, you will need to export the data.|

Reviewer: ```suggestion
 |9.<br/>|You can also export the report data into an Excel .csv file by selecting the **Export** link. This exports data for all users and enables you to do simple aggregation, sorting, and filtering for further analysis. If you have less than 100 users, you can sort and filter within the table in the report itself. If you have more than 100 users, in order to filter and sort, you will need to export the data.|
```
